### DISCUSSION TITLE
Audible just discontinued their app for Windows.  They're forcing us to use phones.

### DISCUSSION THREAD

**Discussion by wjabrams** [audiamus/AaxAudioConverter#109] [Q&A]
Audible just discontinued their app for Windows.  They're forcing us to use phones.  Any advice on how to extract an aax file from an iphone.  I have over 700  titles.  They're killing me.  Thanks, Amazon!

**Comment by Sailfish** [comment]
I'm still able to download titles from the Win10 Audible app Library and play them on Win10 using the app. After I download a title it is located as a *.aax in the Downloads folder. Could you be referring to the AudibleSync app (which does seem to be inoperative)?

**Comment by audiamus** [comment]
> I'm still able to download titles from the Win10 Audible app Library and play them on Win10 using the app. 

They say over here that you will only be able to use the Win10 store app until July.

> Any advice on how to extract an aax file from an iphone. 

Yes. We have to emulate a phone. And the open source community has found out how to do that. My own implementation is called [Book Lib Connect](https://github.com/audiamus/BookLibConnect), coincidentally released in a first version just last Sunday. _It has a bug, though. If you have any book series title, where the sequence number is not a number, it will fail. I will fix that._  

Book Lib Connect is a standalone app at the moment, but the long term plan is to integrate its functionality into AAX Audio Converter.

**Comment by wjabrams** [comment]
GREAT!!!  Thanks for keeping one step ahead of them. I don’t know how Audible still has customers. You have to fight them to get the product they’re selling. Great work. It’s much appreciated.
